Package.swift

```diff
@@ -5,6 +5,9 @@ import PackageDescription
 
 let package = Package(
   name: "FlexLayout",
+  platforms: [
+    .iOS(.v12)
+  ],
```

Comment:
It seems that the settings were removed in this commit. If there’s any issue I’m not aware of, please let me know!

https://github.com/layoutBox/FlexLayout/commit/f8227b3bb9bc2cf7676ff836d458a7a184f1da7f